src/main/Header.tsx

```diff
@@ -112,6 +124,7 @@ const Logo: React.FC = () => {
 
   const { t } = useTranslation();
   const { scheme } = useColorScheme();
+  const { width } = useWindowDimensions();
```

Comment:
Do we need this? Else this line and the useWindowDimensions() import can be removed.

Reply:
Removed it, thanks for catching that.